Makefile

```
@@ -4,7 +4,7 @@ export ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 DISK?=$(shell ls $(ROOT_DIR)/build/*.qcow2 2> /dev/null)
 DISKSIZE?=20G
 ISO?=$(shell ls $(ROOT_DIR)/build/*.iso 2> /dev/null)
-FLAVOR?=green
+FLAVOR?=green-rpi
```

I would keep the default flavor as green, then it can be overridden when building for rpi.

```suggestion
FLAVOR?=green
```